src/lib/ui/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -116,6 +117,13 @@ function AccessCodeRow(props: {
   onClick: () => void
 }) {
   const { onClick, accessCode } = props
+
+  const errorCount = accessCode.errors?.length ?? 0
+  const isPlural = errorCount === 0 || errorCount > 1
+  const issueIconTitle = isPlural
+    ? `${errorCount} code issues`
+    : `${errorCount} code issue`
```

Comment:
Can you move this to the `t` object below?

Reply:
Not sure about the new error, but ref https://github.com/seamapi/react/blob/e7a2713fc43add75cb08d3e8ed0956426ab758da/src/lib/ui/DeviceTable/DeviceHealthBar.tsx#L29

Reply:
(Mike considered / handled the 0 case which was non-obvious!)

Reply:
Updated in [10b2663](https://github.com/seamapi/react/pull/196/commits/10b2663a33ea31179fe409fcd2f61401df5c4a9c) (the 0 case was handled previously)